### DISCUSSION TITLE
Addition of Cava visualiser on AGS bar and/or media widget

### DISCUSSION THREAD

**Discussion by sametaor** [end-4/dots-hyprland#710] [Ideas]
Hi! I'm a huge fan of the project, as it is slowly helping me know how AGS can be used in a plethora of ways. I was wondering whether it'd be possible for the Cava visualiser to be added onto the AGS bar inside the media title group, as well inside the media widget, perhaps serving as a background for it or just being shown on demand. Moreover, it'd be better if all of this is toggleable, as usual.
If this is already possible with a few tweaks, please do let me know about that as well.

Thanks!

**Comment by end-4** [comment]
Not just "a few tweaks" unfortunately
But you might wanna check ferreo's dotfiles which has a cava widget: https://github.com/ferrreo/ags-dots
He said his cava service can be paused to prevent unecessary power consumption, but idk about that. I just knew the widget worked back when I tried it

Note that "back when I tried it" is WAY back so I can't guarantee the AGS config will work. The ~~Go~~ Zig program (my bad, cava one is Zig while other programs are Go) for listening to cava should be fine though.

**Reply by sametaor** [reply]
> Not just "a few tweaks" unfortunately But you might wanna check ferreo's dotfiles which has a cava widget: [ferrreo/ags-dots](https://github.com/ferrreo/ags-dots) He said his cava service can be paused to prevent unecessary power consumption, but idk about that. I just knew the widget worked back when I tried it
> 
> Note that "back when I tried it" is WAY back so I can't guarantee the AGS config will work. The ~Go~ Zig program (my bad, cava one is Zig while other programs are Go) for listening to cava should be fine though.

Alright, thanks a lot! Will look into it 👍
